test/unit/lib/formatter/index.js

```diff
@@ -218,6 +218,18 @@ describe('Formatter', function () {
 
       expect(reformatted).to.deep.equal(options);
     });
+
+    it('can unformat not format value', function () {
+      var formatter = new Formatter('{{9999}} {{9999}}');
+      var options = {
+        selection: {start: 6, end: 6},
+        value: '123456'
+      };
+
+      var unformatted = formatter.unformat(options);
```

Comment:
For clarity, can we include an assertion about what the formatted value looks like?

Reply:
> For clarity, can we include an assertion about what the formatted value looks like?

Modified the test, let me know if I did not what you planned